README.md

```diff
@@ -220,7 +220,3 @@ To build `MapboxDirectionsCLI` using SPM:
 
 1. `swift build`
 1. `swift run MapboxDirectionsCLI -h` to see usage.
-
-## Pricing
-
-API calls made to the Directions API are individually billed by request. Review the [pricing information](https://docs.mapbox.com/api/navigation/directions/#directions-api-pricing) and the [pricing page](https://www.mapbox.com/pricing/#directions) for current rates.
```

Comment:
This part shouldn’t be removed. We expect it to no longer be part of the diff after rebasing, but it should remain in the codebase because it’s now in release-v2.0.